src/components/editions/headline/index.tsx

```diff
@@ -42,6 +42,11 @@ const commentStyles = css`
 	padding-right: 6rem;
 `;
 
+const lStyles = css`
+	padding-bottom: 0;
+	padding-right: 6rem;
+`;
+
```

Comment:
Doesn't look like this is being used, can we remove it?

Reply:
Good spot 👍